sentry-core/src/main/java/io/sentry/core/SessionAdapter.java

```diff
@@ -182,6 +175,10 @@ public Session read(JsonReader reader) throws IOException {
     }
     reader.endObject();
 
+    if (status == null || started == null || release == null || release.isEmpty()) {
```

Comment:
l: Maybe we log here? Would help debugging if this happens

Reply:
good idea